src/me/StevenLawson/TotalFreedomMod/Commands/Command_opall.java

```diff
@@ -24,12 +24,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
                 doSetGamemode = true;
                 targetGamemode = GameMode.CREATIVE;
             }
-            else if (args[0].equals("-s"))
```

Comment:
Not sure why you removed this to replace it with a permissions check. Surly you would be better off leaving this here and performing the permissions check within this IF statement. That way if they are not senior you can easily throw a no perms error, where here you are not able / have done this... Just makes more work for yourself this way around.